index.html

```diff
@@ -16,7 +16,7 @@
  <!--My name in every page-->
     <div class="fixed">
         <h1>
-            Igneri Illa The Great
+            Igneri Illa The Great, but Drake's greater
```

Comment:
this doesn't seem right, take another look

Reply:
I don't like the contraction
```suggestion
            Igneri Illa The Great, but Drake is greater
```